.github/workflows/slsa_for_ml.yml

```diff
@@ -47,7 +47,6 @@ jobs:
     strategy:
       fail-fast: false # Don't cancel other jobs if one fails
       matrix:
-        os: [ubuntu-latest] ## , macos-latest, windows-latest]
```

Comment:
Please keep this, as it's a reminder that we'd need to re-enable the jobs for those OSes when we get back to fixing the SLSA part.

Reply:
I thought the include comments would be enough, I didn't remove them... reverting anyway but it's confusing to have two sets of configuration and comments.